engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -276,9 +307,15 @@ def scc_activate_product
             end
             logger.info "Product #{@product.product_string} successfully activated with SCC"
             InstanceVerification.update_cache(request.remote_ip, @system.login, @product.id)
+            # if the system was PAYG and the registration code is valid for the extension,
```

Comment:
s/was/is/

Reply:
The comment suggest that we are checking the validity of the registration code. I do not see such a check.

Reply:
I think the check is there ? in line 300 `response = SccProxy.scc_activate_product(@product, auth, params, mode)`

the SCC endpoint for activating a product with a regcode/token performs the check, so if we reach that point, regcode/token is validation is OK